setup-nexus-npm/README.md

```diff
@@ -1,22 +1,20 @@
 # setup-nexus-npm
 
-Configure a github action to point towards a self hosted Npm repository. After usage, any calls to `npm install` will point towards the repository at the endpoint provided.
+A composite github action for creating a .npmrc file that will point to the npm repositories in nexus
 
 ## Parameters
-- Endpoint: the npm registry endpoint
-- Login: the login for the self hosted npm repository, typically stored as a secret
-- Password: the password for the self hosted npm repository, typically stored as a secret
+- username: the username for accessing the npm repositories in nexus, typically provided as a secret from the calling workflow
```

Comment:
Based on what you mentioned in your learnings, perhaps the wording here should say "...must be provided..." if that is the only way that this will work.  If not, please ignore!  :)

Reply:
added "Required"